### DISCUSSION TITLE
Image enhancing settings

### DISCUSSION THREAD

**Discussion by Cremz** [ImageMagick/ImageMagick#2520] [Help]
I'm looking for some help processing images.
Can anyone tell me what settings i would need to use to obtain the effect in [this picture](https://imgur.com/a/AZlc4te)?
I'm pretty sure it's done though imagemagick, please tell me this is possible and it's not just Photoshopped by hand :)

**Comment by snibgo** [comment]
What effect do you mean? I suspect the skin has been smoothed, and there is haloing and staircasing from sharpening. I can't see any other obvious effect.

Without seeing the original unprocessed image, I can't say what settings are needed.

**Reply by Cremz** [reply]
Sorry, I should have been more clear. In the link there are two images. A large, original image and a smaller, processed thumbnail version.
I know how to resize and crop, i know about sharpening, but all the settings i'm trying do not achieve the same skin smoothing, and sharp colors.

**Comment by snibgo** [comment]
I was describing the large image. It has haloing and staircasing from sharpening. It may also have skin smoothing, though that might be physical makeup.

The small image is lighter, and resized and cropped.The small image might have been made from the large version, or both versions might have been made from some other image.

If the small was made from the large, then I suspect it is also sharpened more (creating more halos) but I can't be sure. I can't see evidence of more skin smoothing.

EDIT: the small image was not made from the large image. In the light area on the left, the large image is burnt out at 100% white, but the small image has detail.

So both images were made from some other image.
